docs/migration-guide.md

```diff
@@ -30,6 +30,7 @@ The type `sds` is removed from the public API.
 ### Removed API functions
 
 * `redisClusterSetMaxRedirect` removed and replaced with `valkeyClusterSetOptionMaxRetry`.
+* `redisClusterSetOptionAddNode` removed and replaced with `valkeyClusterSetOptionAddNodes`.
```

Comment:
This text is in the migration guide for how a user can replace `hiredis-cluster` with `libvalkey`, and under the headline `Removed API functions`.
The precursor in the doc is "The general actions needed are: Replace the prefix `redis` with `valkey` in API usages." and then this aimed to note that the API **redis**ClusterSetOptionAddNode is removed, i.e. the prefix cant simply be changed.

This was aimed to help when doing this replacement, but it can certainly be improved.
Any suggestions?

Reply:
@mkmkme I think you missed the "s" in the end. It's easy to miss.

* redisClusterSetOptionAddNodes renamed
* redisClusterSetOptionAddNode was removed

Reply:
@bjosv If we can clarify it in some way to avoid confusing users, please do.

Reply:
```suggestion
* `redisClusterSetOptionAddNode` removed and replaced with `valkeyClusterSetOptionAddNodes`.
  (Note the "s" in the end of the function name.)
```